frontend/wallet/src/render/components/Alert.re

```diff
@@ -15,7 +17,8 @@ module Styles = {
       ),
     ]);
 
-  let text = Theme.Text.Body.semiBold;
+  let text =
```

Comment:
You'll notice that there's a growing amount of `textTransform(`capitalize)`

This requires a larger discussion. For i18n work, I usually lowercase everything. And then let CSS handle the actual rendering. Reactintl `id`'s should be lowercased, but there's the argument that the message and defaultMessage themselves dont necessarily have to be lowercased.

Up to your team to determine what's best. The downside of lowercasing everything is having to titlecase or uppercase things manually when a certain component may not have the ability to style the text easily.